pkg/cmd/server/origin/controller/serviceaccount.go

```diff
@@ -0,0 +1,103 @@
+package controller
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/golang/glog"
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	"github.com/openshift/origin/pkg/cmd/server/crypto"
+
+	"k8s.io/client-go/util/cert"
+	kapiv1 "k8s.io/kubernetes/pkg/api/v1"
+	sacontroller "k8s.io/kubernetes/pkg/controller/serviceaccount"
+	"k8s.io/kubernetes/pkg/serviceaccount"
+)
+
+type ServiceAccountControllerOptions struct {
+	ManagedNames []string
+}
+
+func (c *ServiceAccountControllerOptions) RunController(ctx ControllerContext) (bool, error) {
+	if len(c.ManagedNames) == 0 {
+		return true, fmt.Errorf("Skipped starting Service Account Manager, no managed names specified")
+	}
+
+	options := sacontroller.DefaultServiceAccountsControllerOptions()
+	options.ServiceAccounts = []kapiv1.ServiceAccount{}
+
+	for _, saName := range c.ManagedNames {
+		sa := kapiv1.ServiceAccount{}
+		sa.Name = saName
+
+		options.ServiceAccounts = append(options.ServiceAccounts, sa)
+	}
+
+	//REBASE: add new args to NewServiceAccountsController
```

Comment:
@ncdc addded this in Nov/2016... if this refers to args `options` then maybe the comment is still valid.